hail/python/hail/vds/combiner/variant_dataset_combiner.py

```diff
@@ -30,14 +29,17 @@ class VDSMetadata:
         Number of samples contained within the Variant Dataset at `path`.
 
     """
+    path: str
+    n_samples: int
 
-    def __init__(self, path: str, n_samples: int):
-        self.path = path
-        self.n_samples = n_samples
 
-    def to_tuple(self) -> Tuple[str, int]:
-        """Convert to a serializable representation."""
-        return (self.path, self.n_samples)
+class VDSType(NamedTuple):
```

Comment:
I feel like VDS being a type makes it somehow official.

Reply:
fair enough, renamed